docs/kustomize.md

```diff
@@ -1,50 +0,0 @@
-# Deploy with Kustomize
```

Comment:
💡 Can we please keep this as a document, will be good to update the instructions for newbieusers who want to see how `kustomize` file will fit together and how they can consume this in their tool or their purpose. Thanks

cc: @rzhang628 and @qpetraroia

Reply:
documentation is in the example file we added. This brings no value so we should remove it